Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -781,7 +781,21 @@ private extension PurchasesOrchestrator {
                                              maximumRetries: Self.receiptRetryCount,
                                              sleepDuration: Self.receiptRetrySleepDuration)
         } else {
+            // See https://github.com/RevenueCat/purchases-ios/pull/2245 and
+            // https://github.com/RevenueCat/purchases-ios/issues/2260
+            // - Release or production builds:
+            //      We don't _want_ to always refresh receipts to avoid throttling errors
+            //      We don't _need_ to because the receipt will be refreshed by the backend using /verifyReceipt
+            // - Debug and sandbox builds (potentially using StoreKit config files):
+            //      We need to always refresh the receipt because the backend might not use /verifyReceipt
```

Comment:
let's specify that the backend will not use `/verifyReceipt` when the receipt was generated locally with StoreKitTest / StoreKit Configuration files

Reply:
Done.